tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -0,0 +1,126 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run integration tests for mounted directory.
+# $1 testbucket.
```

Comment:
you can define some useful name here and then use in rest of places e.g.
TEST_BUCKET_NAME=$1
MOUNT_DIR=$2